content/Solutions/Optimizations/Security.md

```diff
@@ -1,20 +1,36 @@
 ---
 title: "Security Recommendations"
-description: "Best practices when tuning TrueNAS to minimize security vulnerabilities."
+description: "Best practices to administrate TrueNAS securely."
 weight: 30
 aliases:
   - /core/solutions/optimizations/security/
 tags:
  - coressh
  - scalessh
+keywords:
+ - TrueNAS Security
 ---
 
 {{< toc >}}
 
-When using services on TrueNAS, especially services that allow outside connections, there are some best practices to follow to ensure your system is safe and secure.
-Several different system services are disscused in this article.
+Follow these best practices to administrate TrueNAS securely.
 
-## iSCSI
+## General Recommendations
+
+1. Specially modifying the base TrueNAS firmware image is unsupported and can create security issues.
+2. Keep TrueNAS up to date with the most recent updates for your supported version.
+3. Upgrade to new major releases in a timely manner consistent with the deployment use case.
+4. Restrict new TrueNAS user accounts to the most minimal set of permissions and access possible.
+5. Never grant TrueNAS user accounts (local or Directory Services added accounts) access to SSH or console shells unless that account is explicitly trusted.
+6. Use complex passwords and Two-Factor Authentication (2FA) for all TrueNAS Root and Administrator accounts.
+7. Disable any Network services not in use.
+8. Restrict the TrueNAS web, IPMI, and any other management interfaces to private subnets away from untrusted users.
+9. Review any plugin, App, or Virtual Machine (VM) deployment scenario for additional security exposure or vulnerabilities.
+    iXsystems cannot resolve security vulnerabilities introduced from within user-deployed virtualized environments.
+
+## Hardening Specific System Services
+
+### iSCSI
 
 Follow the iSCSI creation wizard unless a specific configuration is required.
 To create an iSCSI share, go to **Sharing > Block Shares (iSCSI)** and click *WIZARD*.
```

Comment:
line 41 the option to add should be in bold not italics.
Line 44 Initiators and Authorized Networks should be bold not italics.